crypto/chacha20poly1305/chacha_merged.c

```diff
@@ -23,8 +23,8 @@ void ECRYPT_init(void)
   return;
 }
 
-static const char sigma[16] = "expand 32-byte k";
-static const char tau[16] = "expand 16-byte k";
+static const char sigma[16] __attribute__((nonstring)) = "expand 32-byte k";
+static const char tau[16] __attribute__((nonstring)) = "expand 16-byte k";
```

Comment:
This a valid solution, but why not use the fully portable solution which they used in https://github.com/bitcoin-core/secp256k1/blob/36e76952cbf1cf54ddd2d8756cc31a486e2ba1d9/src/modules/schnorrsig/main_impl.h#L48?
```suggestion
static const char sigma[] = {'e', 'x', 'p', 'a', 'n', 'd', ' ', '3', '2', '-', 'b', 'y', 't', 'e', ' ', 'k'};
static const char tau[] = {'e', 'x', 'p', 'a', 'n', 'd', ' ', '1', '6', '-', 'b', 'y', 't', 'e', ' ', 'k'};
```

Reply:
Sounds good - fixed in 50e47f3059.